Sources/Down/AST/Styling/Stylers/DownStyler.swift

```diff
@@ -218,6 +219,28 @@ open class DownStyler: Styler {
             .link: url,
             .foregroundColor: colors.link])
     }
+    
+    private func styleGenericImg(in str: NSMutableAttributedString, url: URL) {
+        let semaphore = DispatchSemaphore(value: 0)
+        URLSession.shared.dataTask(with: url) { data, _, _ in
+            if let data = data {
```

Comment:
A couple things:

1. What happens when the url doesn't point to an image?
2. What happens when there is a network error?
3. I think it's important to make image fetching an optional behavior (preferably opt in) since users of the library may not want to automatically fetch data from unknown urls. This could be achieved with a simple flag in the styler configuration.